src/win32.mak

```
@@ -334,13 +332,8 @@ detab:
 tolf:
 	$(TOLF) $(SRCS) $(GLUESRC) $(ROOTSRC) $(BACKSRC) $(MAKEFILES)
 
-zip: detab tolf $(MAKEFILES)
-	$(DEL) dmdsrc.zip
-	$(ZIP) dmdsrc $(MAKEFILES)
-	$(ZIP) dmdsrc $(SRCS)
-	$(ZIP) dmdsrc $(GLUESRC)
-	$(ZIP) dmdsrc $(BACKSRC)
-	$(ZIP) dmdsrc $(ROOTSRC)
+zip: $(GEN)\build.exe
```

Missing dependencies:
```suggestion
zip: detab tolf $(GEN)\build.exe
```